Actions/AL-Go-Helper.ps1

```diff
@@ -1841,6 +1841,13 @@ function CreateDevEnv {
             $sharedFolder = $baseFolder
         }
 
+        # If UseCompilerFolder is set, set the parameter on Run-AlPipeline
```

Comment:
Just add "useCompilerFolder" to the list of properties to "move" in line 1826 instead.